hawc/apps/assessment/templates/assessment/assessmentvalue_form.html

```diff
@@ -3,7 +3,8 @@
 {% load crispy_forms_tags %}
 
 {% block content %}
-<div class="d-flex justify-content-end">
+<div class="d-flex justify-content-between">
+    <h3>Assessment Value for {{ assessment }}</h3>
```

Comment:
Just for consistency with other forms we typically do at the form-level. It might be better to add here: https://github.com/shapiromatron/hawc/blob/bfb056dffb26bfe9e66191ae254fe10b18e725c6/hawc/apps/assessment/forms.py#L254-L267


In addition, I think we'll need the same for the AssessmentDetails: https://github.com/shapiromatron/hawc/blob/bfb056dffb26bfe9e66191ae254fe10b18e725c6/hawc/apps/assessment/forms.py#L160

I think it should be as simple as adding `legend_text` for the update fields.